src/System Application/Test/EntityText/src/MktTextAccuracyAIT.Codeunit.al

```diff
@@ -0,0 +1,240 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.EntityText;
+
+using System.TestTools.AITestToolkit;
+using System.Text;
+
+codeunit 134640 "Mkt Text Accuracy AIT"
+{
+    Subtype = Test;
+
+    [Test]
+    procedure TaglineParagraphInspiring()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        Category: Text;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputPrompt := AITestContext.GetInput().ValueAsText();
+        Facts := LineInputToDictionary(InputPrompt);
+        Format := Enum::"Entity Text Format"::TaglineParagraph;
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Inspiring, Format, Enum::"Entity Text Emphasis"::None);
+
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts, Category), Category, Response, Enum::"Entity Text Tone"::Inspiring, Format));
+    end;
+
+    [Test]
```

Comment:
Tests are missing comments:
[Scenario]
[Given]
[When]
[Then]

Did we stop using these?